llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -610,6 +599,15 @@ def IREELinalgExt_AttentionOp : IREELinalgExt_Op<"attention",
     ShapedType getOutputType() {
       return getOutput().getType().cast<ShapedType>();
     }
+    ShapedType getMaxType() {
```

Comment:
Make this also `std::optional<ShapedType>`. So if `getMax()` return `std::nullopt` then this method return `nullopt` too.